x-pack/plugins/enterprise_search/server/lib/engines/field_capabilities.test.ts

```diff
@@ -29,24 +29,940 @@ describe('engines field_capabilities', () => {
     jest.clearAllMocks();
   });
 
-  it('gets engine alias field capabilities', async () => {
-    const fieldCapsResponse = {} as FieldCapsResponse;
+  describe('fetchEngineFieldCapabilities', () => {
```

Comment:
Nitpick: We should try to move away from naming things `*engines*`. I'd be OK if we decided to do this in one separate rename/refactor PR though.